website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -24,7 +24,7 @@ Admin partitions exist a level above namespaces in the identity hierarchy. They
 
 Each Consul cluster will have at least one default admin partition (named `default`). Any resource created without specifying an admin partition will inherit the partition of the ACL token.
 
-The `default` admin partition is special in that it may contain namespaces and other entities that are replicated between datacenters.
+The `default` admin partition is special in that it may contain namespaces and other entities that are replicated between datacenters. The `default` partition should also contain the Consul servers.  
```

Comment:
Servers can only exist in the default partition
```suggestion
The `default` admin partition is special in that it may contain namespaces and other entities that are replicated between datacenters. The `default` partition must also contain the Consul servers.  
```